### ISSUE TITLE
aligning slowly

### ISSUE THREAD

**Issue by wotulong** [keithito/tacotron#79]
My training data is LJSpeech, and I change the params batch_size=24 , init_lr = 0.0018, and keep other params unchanged. And my result below:
![step-164000-align](https://user-images.githubusercontent.com/4609486/33475876-ec83cdce-d6ba-11e7-9946-b455d4da8da1.png)
What can I do to improve it.
thanks!

**Comment by keithito** [comment]
That's odd. Those hyperparameter changes seem very minor, and I would not have expected them to affect the speed at which the alignment is learned. What does your loss curve look like in Tensorboard?

**Comment by wotulong** [comment]
I have change the dataset of Nancy, and I have delete the log files before.Sorry about that.The result is good with Nancy. 
![image](https://user-images.githubusercontent.com/4609486/33877409-3cc4c88c-df63-11e7-98d7-178623ae86c8.png)
Thanks for your reply.
